src/main/java/edu/rosehulman/quota/Database.java

```diff
@@ -130,6 +130,10 @@ public Quota getQuota(String partnerId, String productId, String userId, String
 		return null;
 		//return partnerMap.get(partnerId).getProduct(productId).getUser(userId).getQuota(quotaId);
 	}
-	
+
+	public boolean addUser(String partnerId, String productId, String userId) {
+	  Product product = this.partnerMap.get(partnerId).getProduct(productId);
+    return product.addUser(new User(userId, product));
```

Comment:
You can just return null on this for now (because we mock this method in testing). We do not save users on product objects, we save them into the database and then when we do product.getUser() we call that database. For the next milestone, we will connect to the database here and insert it.